packages/containers/src/GuidedTour/GuidedTour.container.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
+import { cmfConnect } from '@talend/react-cmf';
+import GuidedTour from '@talend/react-components/lib/GuidedTour';
+
+class GuidedTourContainer extends React.Component {
+	static displayName = 'Container(GuidedTour)';
+
+	static propTypes = {
+		...cmfConnect.propTypes,
+		show: PropTypes.bool,
+		steps: PropTypes.oneOf([PropTypes.array, PropTypes.func]),
+		onClose: PropTypes.func,
+	};
+
+	state = {
+		controls: true,
```

Comment:
I would go either full cmf state, or full non-cmf state, not a mix. It can become confusing

Reply:
My thought was that`controls` is only for container scope rather than `open` is controlled from outside

Reply:
In that case, we can move that in the component. It won't be pure, but I don't think it's a pb anymore.
By moving controls in the component, the step generation from props will move too. But that will make the component easier to use in non cmf apps